src/js/stringoperation.js

```diff
@@ -0,0 +1,9 @@
+const Strings = (() => {
+  const capitalize = (str) => str.toUpperCase();
```

Comment:
Capitalize should make only the first character capital ![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/capitalize_requirement.png)